doc/modules/model_evaluation.rst

```diff
@@ -61,7 +61,7 @@ Scoring                           Function
 'accuracy'                        :func:`metrics.accuracy_score`
 'balanced_accuracy'               :func:`metrics.balanced_accuracy_score`
 'average_precision'               :func:`metrics.average_precision_score`
-'brier_score_loss'                :func:`metrics.brier_score_loss`
+'neg_brier_score_loss'            :func:`metrics.brier_score_loss`
```

Comment:
I think 'neg_brier_score` might be a more friendly name. neg already implies that the score would otherwise decrease with improvement, and "Brier score loss" is not something known in the literature: it appears 70 times on Google not associated with scikit-learn.

Reply:
> I think 'neg_brier_score` might be a more friendly name. neg already implies that the score would otherwise decrease with improvement,

Maybe neg_brier_score_loss is a better name? because the name of the function is ``brier_score_loss`` and we also have other similar scorers like "neg_log_loss"? @jnothman

> and "Brier score loss" is not something known in the literature

If so, we should change the name of the function.

Reply:
Can you explain the definition of this neg_brier_score? look at my post please: https://stackoverflow.com/questions/62853440/neg-brier-score-returns-negative-score